Updates related to three MinIO Client Dec 2023 releases (#1098)
minio/docs: source/reference/minio-mc-admin/mc-admin-replicate.rst

            mc admin replicate add minio1 minio5

>          The following command creates a new site replication that includes the peer sites ``minio1``, ``minio2``, and ``minio3``, and enables the replication of ILM expiration rules across sites.

Reviewer: ```suggestion
         The following command creates a new site replication configuration with ILM expiration rule synchronization between peer sites ``minio1``, ``minio2``, and ``minio3``.
```

Author: done

      The deployment ID can be found by running :mc-cmd:`mc admin replicate info ALIAS`

   .. mc-cmd:: --disable-ilm-expiry-replication
      :optional:

      .. versionadded:: mc RELEASE.2023-12-02T02-03-28Z

>       Stops the replication of ILM expiration rules between peer sites.

Reviewer: Wonder if we need a note here that this does *not* remove any ILM expiration rules already synchronized between sites.

Author: Added a sentence to this effect.

      Display the replication status of a specific group by including the group name after the flag.

   .. mc-cmd:: --ilm-expiry-rules
      :optional:

      .. versionadded:: mc RELEASE.2023-12-02T02-03-28Z

      Display sync information about ILM expiration rules.

>    .. mc-cmd:: --ilm expiry rule

Reviewer: ```suggestion
   .. mc-cmd:: --ilm-expiry-rule
```

Author: Good catch! Fixed.

      :optional:

      .. versionadded:: mc RELEASE.2023-12-02T02-03-28Z

      Display replication status information about the specified ILM expiration rule.

> 

Reviewer: Are these mutually exclusive? I would assume `--ilm-expiry-rules` would stomp `--ilm-expiry-rule`

Author: Yes. Added info.
